src/components/AllQuests.jsx

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import getQuests from "../modules/getQuests";
+import QuestCard from "./QuestCard";
+import { Grid } from "semantic-ui-react";
+
+const AllQuests = () => {
+  const dispatch = useDispatch();
+  const quests = useSelector((state) => {
+    return state.quests.quests;
```

Comment:
You don't have to do this on a second line if you use `(` instead of `{`. Also, we change naming of all 'quests' to 'requests'. This applies to the whole PR

Reply:
Tip: do search and replace ....

Reply:
done